src/main/java/me/totalfreedom/totalfreedommod/ChatManager.java

```diff
@@ -113,7 +113,17 @@ public void adminChat(CommandSender sender, String message)
             }
         }
     }
-
+    
+    public void adminHelp(Player sender, String message) {
+       for (Player player : server.getOnlinePlayers())
+        {
+            if (plugin.al.isAdmin(player))
+            {
+                playerMsg(player, ChatColor.RED + "[ADMINHELP] " + ChatColor.GOLD + sender.getName() + ": " + message);
```

Comment:
Im concerned that the gold may get confused with the new admin chat colour. Can this be changed to another colour? Dark Blue or a light red maybe?